src/pages/en/guides/styling.md

```diff
@@ -114,13 +101,57 @@ You can achieve the same by using the `:global()` function at the root of a sele
 </style>
 ```
 
-It’s recommended to only use this in scenarios where a `<link>` tag won’t work. It’s harder to track down errant global styles when they’re scattered around and not in a central CSS file.
+> ⚠️ These global styles apply throughout your entire project! 
+
+It may be easy to lose track of which Astro component is defining styles globally, and harder to troubleshoot errant global styles when they’re scattered around and not in a central CSS file. So, this method is recommended only when you cannot otherwise apply global styling via an import or `<link>`.
+
+
+## External Styles
+
+There are two ways to resolve global stylesheets: an ESM import for files located within your project source, and an absolute URL link for files in your /public directory, or hosted outside of your project.
 
-📚 Read our full guide on [Astro component syntax][astro-component] to learn more about using the `<style>` tag.
 
-### Variables in Scripts & Styles
+### Import a Global Stylesheet
 
-In Astro v0.21+, _serializable_ server-side variables can be passed into client-side `<style>` or `<script>`.
+Astro detects these CSS imports and then builds, optimizes, and adds the CSS to the page automatically. 
+
+Import a global stylesheet **at the top** of an Astro component script, using its relative file path, along with any other imports:
+
+```astro
+---
+// Astro will include and optimize this CSS for you automatically
+// This also works for preprocessor files like .scss, .styl, etc.
+import '../styles/utils.css';
+
+const title="My Astro Page"
+---
+<html><!-- Your page here --></html>
+```
+
+Importing CSS files should work anywhere that ESM imports are supported, including:
+- JavaScript files
+- TypeScript files
+- Astro component front matter
+- non-Astro components like React, Svelte, and others
+
+When a CSS file is imported using this method, any `@import` statements are also resolved and inlined into the imported CSS file.
+
+### Stylesheet Link
+
+When your stylesheet is located in your `/public` directory, or when using a public stylesheet hosted offsite (e.g. a Prism theme), use a `<link>` tag with an absolute URL reference.
+
+```html
+<head>
+  <link rel="stylesheet" href="/styles/global.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism-tomorrow.css">
+</head>
+```
+
+This approach skips the CSS processing, bundling and optimizations that are provided by Astro when you use the `import` method described above. These files will not be transformed.
+
+## Variables in Styles
+
+In Astro v0.21+, _serializable_ server-side variables can be passed into client-side `<style>`.
```

Comment:
Is this an opportunity to use `<Since>` 👀 

```suggestion
<Since v="0.21.0" />

_Serializable_ server-side variables can be passed into client-side `<style>`.
```